digid_eherkenning/oidc/models/eherkenning.py

```diff
@@ -40,49 +76,64 @@ class Meta:
-        max_length=50,
+class EHerkenningBewindvoeringConfig(AuthorizeeMixin, OpenIDConnectBaseConfig):
+    # XXX: how do we determine the identifier type?
+    representee_claim = ClaimField(
+        verbose_name=_("representee identifier claim"),
+        # TODO: this is Anoigo, but could really be anything...
+        default=ClaimFieldDefault("sel_uid"),
         help_text=_(
-            "Name of the claim in which the KVK of the company being represented is stored"
+            "Name of the claim holding the identifier (like a BSN, RSIN or CoC number) "
+            "of the represented person/company."
         ),
     )
-    gemachtigde_person_claim_name = models.CharField(
-        verbose_name=_("gemachtigde person claim name"),
-        default="gemachtigde.pseudoID",
-        max_length=50,
+
+    mandate_service_id_claim = ClaimField(
+        verbose_name=_("service ID claim"),
+        default=ClaimFieldDefault("urn:etoegang:core:ServiceID"),
         help_text=_(
-            "Name of the claim in which the ID of the person representing a company is stored"
+            "Name of the claim holding the service ID for which the company "
+            "is authorized."
         ),
     )
+    mandate_service_uuid_claim = ClaimField(
+        verbose_name=_("service UUID claim"),
+        default=ClaimFieldDefault("urn:etoegang:core:ServiceUUID"),
+        help_text=_(
+            "Name of the claim holding the service UUID for which the company "
+            "is authorized."
+        ),
+    )
+
     oidc_rp_scopes_list = ArrayField(
         verbose_name=_("OpenID Connect scopes"),
         base_field=models.CharField(_("OpenID Connect scope"), max_length=50),
         default=get_default_scopes_bsn,
         blank=True,
         help_text=_(
             "OpenID Connect scopes that are requested during login. "
-            "These scopes are hardcoded and must be supported by the identity provider"
+            "These scopes are hardcoded and must be supported by the identity provider."
         ),
     )
 
     class Meta:
         verbose_name = _("OpenID Connect configuration for eHerkenning Bewindvoering")
 
     @property
-    def digid_eherkenning_machtigen_claims(self) -> dict[str, ClaimPath]:
-        # TODO: this nomenclature isn't entirely correct
+    def mandate_claims(self) -> dict[str, ClaimPath]:
         return {
-            "vertegenwoordigde": [self.vertegenwoordigde_company_claim_name],
-            "gemachtigde": [self.gemachtigde_person_claim_name],
+            "representee": self.representee_claim,
+            # "authorizee_legal_subject_type": self.identifier_type_claim,
+            "authorizee_legal_subject": self.legal_subject_claim,
+            "authorizee_acting_subject": self.acting_subject_claim,
+            # "authorizee_branch_number": self.branch_number_claim,
+            # "service_id": self.mandate_service_id_claim,
+            # "service_uuid": self.mandate_service_uuid_claim,
```

Comment:
the commented out properties can be absent from the claims, which causes issues at the moment in Open Forms.

I expect this to be fleshed out further in the future - I'm also considering moving this part of the code into Open Forms itself since I'm not sure how useful it is at this time to have it here :thinking: